keyboards/citrus/erdnuss65/readme.md

```diff
@@ -0,0 +1,27 @@
+# Erdnuss65
+
+
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Citrus Lab](https://github.com/ctt-t) 
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```

Comment:
These fields should be filled out.

Reply:
Ok, I have changed the file and resubmitted it